docker/docker_transport_test.go

```diff
@@ -150,6 +200,14 @@ func TestReferencePolicyConfigurationNamespaces(t *testing.T) {
 		"docker.io",
 		"*.io",
 	}, ref.PolicyConfigurationNamespaces())
+
+	ref, err = ParseReference("//busybox" + UnknownDigestSuffix)
```

Comment:
If there are 3 cases in the `switch`, they should all be covered by these tests … it’s probably easier to remove the `switch`.

Reply:
Fixed